appengine-java8/datastore/src/test/java/com/example/appengine/MetadataKindsTest.java

```diff
@@ -75,7 +75,7 @@ void printLowercaseKinds(DatastoreService ds, PrintWriter writer) {
     // Start with unrestricted kind query
     Query q = new Query(Entities.KIND_METADATA_KIND);
 
-    List<Filter> subFils = new ArrayList();
+    List<Filter> subFils = new ArrayList<Filter>();
```

Comment:
nitpick. want to use the diamond operator instead?  (applies to multiple occurrences below).

`List<Filter> subFils = new ArrayList<>();`

Reply:
Sure, can use that wherever possible.